src/index.js

```diff
@@ -4,12 +4,11 @@ import ReactDOM from "react-dom";
 import registerTelemetry from "./telemetry";
 import HomeRoot from "./react-components/home-root";
 import InfoDialog from "./react-components/info-dialog.js";
-import queryString from "query-string";
 
-const qs = queryString.parse(location.search);
+const qs = new URLSearchParams(location.search);
 registerTelemetry();
 
 ReactDOM.render(
-  <HomeRoot dialogType={qs.list_signup ? InfoDialog.dialogTypes.updates : null} />,
+  <HomeRoot dialogType={qs.get("list_signup") ? InfoDialog.dialogTypes.updates : null} />,
```

Comment:
this seems fine. I'm not sure where this URL is advertised, but Google has https://hubs.mozilla.com/?list_signup=true indexed:

![image](https://user-images.githubusercontent.com/203725/42977456-22648dc2-8b7c-11e8-99da-5c9d29623ef5.png)

if you want to allow `?list_signup`, you could use `qs.has(…)`. figured would make sense to have parity with the `?room` behaviour. feel free to ignore this.

Reply:
Interesting. I wonder how it got that URL? We should almost certainly take a day to fix up our main page SEO via webmaster tools, sitemap, better meta tags, etc. (I know you have #317 open related to this, supplying OG meta tags, and I intend to go make sure that or similar work gets into master soon.)